packages/server/api/src/app/ai/chat/ai-mcp-chat.controller.ts

```diff
@@ -120,16 +120,10 @@ export const aiMCPChatController: FastifyPluginAsyncTypebox = async (app) => {
       aiConfig,
     });
 
-    const isAnalyticsLoaded = Object.keys(filteredTools ?? {}).some((key) =>
-      key.includes('superset'),
-    );
-    const isTablesLoaded = Object.keys(filteredTools ?? {}).some((key) =>
-      key.includes('table'),
-    );
-
     const systemPrompt = await getMcpSystemPrompt({
```

Comment:
added a new property toolprovider to all the tools, that way its a bit easier to filter now